app/models/pglogical_subscription.rb

```diff
@@ -242,8 +242,22 @@ def find_password
 
   def create_subscription
     MiqRegion.destroy_region(connection, remote_region_number)
+
+    # new_subscription_name also queries the remote, so we fetch it early to avoid a nested remote query
+    subscription = new_subscription_name
+
+    # Creating the slot and subscription enabled in the same command will hang when replicating
+    # between databases in the same cluster. Instead, we'll do the following:
```

Comment:
Can we change this slightly for clarity..something like

```suggestion
    # Creating the slot and subscription enabled in the same command will work when replicating between databases in
    # separate clusters but hang when replicating between databases in the same cluster. Instead, we'll do the following:
```